content/get-started/new-leads/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Set up your own community
+---
+
+There are many communities established within Galaxy based on [**region**](/community/#regional-communities) or [**domain of science**](/community/#communities-of-practice) but if you feel like you just don't see your own or don't feel identified with them, you can create your own community!
+
+Below you'll find a suggested path to follow before you can start your own community, a list of Do's and Dont's and also useful links you should probably check out.
+
+### Path 
+
+#### Learn the basics about Galaxy
+
+First of all, you should  start with our dedicated [Get Started](/get-started/) page to get a head-start of what Galaxy is about. You'll find all the necessary resources and you could even participate in the [Galaxy Mentorship Network](#) to get started with the help of a Galaxy Mentor.
+
+#### Learn about Tool Development
+Galaxy as many tools already installed and ready to be used but when creating your community you'll probably need specific tools for your Galaxy analysis. Don't worry, see our dedicated page for [Tool Authors](/tools/) where you'll find everything you'll need and also reach out to the [Tool Working Group](/community/wg/) via Gitter.
```

Comment:
I'm not sure but I think it's tools wg (plural)

Reply:
I've checked, it is [here](https://github.com/orgs/galaxyproject/projects/13)